README.md

```diff
@@ -35,6 +30,15 @@ npm install
 ## Running Tests
 - npm test to run jest unit test suite for all components 
 
+## CRUD
+
+| Command | Method | Endpoint | Body | Purpose |
+| ------- | ------ | -------- | ---- | ------- |
+| Create | POST | '/:restaurantName/:restaurantID/menu' | JSON | Add new restaurant to platform |
+| Read | GET | '/:restaurantName/:restaurantID/menu' | none | Get data for given restaurant (defined in endpoint) |
+| Update | PUT | '/:restaurantName/:restaurantID/menu' | JSON | Update restaurants information |
+| Delete | DELETE | '/:restaurantName/:restaurantID/menu' | none | Delete restaurant from platform |
+
```

Comment:
I'm a little confused by api info above. Particularly, the "purpose" section. Are you getting/posting/putting/deleting the restaurant? If so, it seems like the endpoint url shouldn't end with "menu"

Reply:
Looking below, it looks like you are doing all those operations on "dishes", not restaurants.

Reply:
You are correct. Has been adapted in my repo. Thanks!